Libraries/LibJS/LocalVariable.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2025, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/FlyString.h>
+
+namespace JS {
+
+struct LocalVariable {
+    FlyString name;
+    bool lexically_declared { false };
```

Comment:
Maybe we should preserve the `DeclarationKind` here. Would use the same amount of space, but could be useful to know whether a local comes from var/let/const :)